modules/apps/remote-app/remote-app-admin-web/src/main/resources/META-INF/resources/display/css/main.css

```diff
@@ -1,4 +1,5 @@
 .portlet-remote-app iframe {
-    min-height: 500px;
-    width: 100%;
+	border: none;
+	min-height: 500px;
+	width: 100%;
```

Comment:
Fun fact: [the original version of the Remote Apps PR](https://github.com/wincent/liferay-portal/pull/365) removed the border in [this commit](https://github.com/wincent/liferay-portal/pull/365/commits/559ad423c75d4aa099ea95c4980f5f0fcd3afc5b).

It then got forwarded [here](https://github.com/liferay-frontend/liferay-portal/pull/270), but the CSS changes somehow got lost in all the rebasing and moving.

This solution is exactly like mine, except I used `border: 0;` to save **AN AMAZING THREE BYTES**.

Reply:
> to save AN AMAZING THREE BYTES.

:rofl: 

I have no opinion or preference about that one, but let me know if you think using `border: 0;` would be better

Reply:
The difference is minuscule, but discussed here: https://stackoverflow.com/questions/2922909/should-i-use-border-none-or-border-0

Reply:
Thanks, after reading that I am going to change it to `border: 0;` - because it is shorter